### DISCUSSION TITLE
unauthorized: authentication required

### DISCUSSION THREAD

**Discussion by skongwong** [netease-youdao/QAnything#255] [Q&A]
执行到一半总是提示没有权限！！！！ 1.8B/3B/7B都一样

[shaokong@localhost QAnything]$ bash ./run.sh -c local -i 0 -b hf -m Qwen-7B-QAnything -t qwen-7b-qanything
From https://github.com/netease-youdao/QAnything
 * branch            master     -> FETCH_HEAD
当前master分支已是最新，无需更新。
请输入您使用的大模型B数(示例：1.8B/3B/7B): 3B
model_size=3B
GPUID1=0, GPUID2=0, device_id=0
llm_api is set to [local]
device_id is set to [0]
runtime_backend is set to [hf]
model_name is set to [Qwen-7B-QAnything]
conv_template is set to [qwen-7b-qanything]
tensor_parallel is set to [1]
gpu_memory_utilization is set to [0.81]
Do you want to use the previous ip: 30.3.66.12? (yes/no) 是否使用上次的ip: ？(yes/no) 回车默认选yes，请输入:yes
Running under native Linux
[+] Running 36/13
 ⠇ etcd 7 layers [⠀⠀⠀⠀⠀⠀⠀]      0B/0B      Pulling                                                               641.7s
 ⠇ qanything_local 46 layers [⣿⣿⣿⣿⠀⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣶⣿⣿⣿⣿⣿⡀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀] 2.208GB/8.257GB Pulling            641.7s
 ✔ elasticsearch 10 layers [⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿]      0B/0B      Pulled                                                   112.9s

unauthorized: authentication required

**Comment by ooooma** [comment]
请问你解决这个问题了吗 我遇到了同样的问题

**Comment by TatyOko28** [comment]
The error unauthorized: authentication required means the script is trying to access a Docker image or resource that requires authentication. This issue can occur due to:

Missing or invalid Docker login credentials.
A private repository being used without proper authentication.
A network configuration issue.
